rca/projects/models.py

```diff
@@ -1,9 +1,222 @@
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    contact_text = models.TextField(blank=True)
+
+    content_panels = BasePage.content_panels + [
+        MultiFieldPanel(
+            [ImageChooserPanel("hero_image"), FieldPanel("hero_colour_option")],
+            heading=_("Hero"),
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel("introduction"),
+                ImageChooserPanel("introduction_image"),
+                FieldPanel("video"),
+                FieldPanel("video_caption"),
+            ],
+            heading=_("Introduction"),
+        ),
+        StreamFieldPanel("body"),
+        StreamFieldPanel("gallery"),
+        StreamFieldPanel("more_information"),
+        MultiFieldPanel(
+            [
+                InlinePanel("project_lead", label="Project team lead", max_num=1),
+                InlinePanel("related_staff", label="Project team"),
+            ],
+            "Project team and staff",
+        ),
+        StreamFieldPanel("partners"),
+        StreamFieldPanel("funders"),
+        StreamFieldPanel("quote_carousel"),
+        MultiFieldPanel(
+            [
+                ImageChooserPanel("contact_image"),
+                FieldPanel("contact_text"),
+                FieldPanel("contact_url"),
+                FieldPanel("contact_email"),
+            ],
+            heading="Contact information",
+        ),
+    ]
+    key_details_panels = [
+        InlinePanel("subjects", label="RCA Experties"),
+        FieldPanel("start_date"),
+        FieldPanel("end_date"),
+        FieldPanel("funding"),
+        DocumentChooserPanel("specification_document"),
+    ]
+
+    edit_handler = TabbedInterface(
+        [
+            ObjectList(content_panels, heading="Content"),
+            ObjectList(key_details_panels, heading="Key details"),
+            ObjectList(BasePage.promote_panels, heading="Promote"),
+            ObjectList(BasePage.settings_panels, heading="Settings"),
+        ]
+    )
+
+    def clean(self):
+        errors = defaultdict(list)
+
+        if self.end_date and self.end_date < self.start_date:
+            errors["end_date"].append(
+                _("Events involving time travel are not supported")
+            )
+        if not self.contact_email and not self.contact_url:
+            errors["contact_url"].append(
+                "Please add a target value for the contact us link"
+            )
+        if self.contact_email and self.contact_url:
+            errors["contact_url"].append(
+                "Only one of URL or an Email value is supported here"
+            )
+
+        if errors:
+            raise ValidationError(errors)
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context["hero_colour"] = "dark"
+        if int(self.hero_colour_option) == LIGHT_TEXT_ON_DARK_IMAGE:
```

Comment:
Isn't `hero_colour_option` an integer field? If so you probably don't need to cast it using `int()`